app/services/support_interface/send_ucas_match_initial_emails_duplicate_applications.rb

```diff
@@ -0,0 +1,28 @@
+module SupportInterface
+  class SendUCASMatchInitialEmailsDuplicateApplications
+    attr_accessor :ucas_match
+
+    def initialize(ucas_match:)
+      self.ucas_match = ucas_match
+    end
+
+    def call
+      return false if ucas_match.initial_emails_sent?
+
+      application_choice = application_choice_for(ucas_match)
+
+      CandidateMailer.ucas_match_initial_email_duplicate_applications(application_choice).deliver_later
+
+      NotificationsList.for(application_choice).each do |provider_user|
+        ProviderMailer.ucas_match_initial_email_duplicate_applications(provider_user, application_choice).deliver_later
+      end
+    end
+
+  private
+
+    def application_choice_for(ucas_match)
```

Comment:
Will this always return the correct application choice? We're sending the specific course details to both provider and candidate — couldn't it just return some other, non-conflicting choice off the form?

Reply:
Afetr discussing this with Aga, a candidate can currently have up to 2 UCAS matches - one for each year. So as long as the current recruitment year applies to the UCAS Match we should be able to retrieve the correct application.

So my approach is incorrect as I followed the pattern I saw in the candidate_mailer for retrieve the application choice when the `application_choice` should be exposed on `UCASMatchedApplication` and retrieved from there, rather than from the candidate (https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/models/ucas_matched_application.rb#L106)

cc @AgaDufrat

Reply:
I think something like this
`ucas_match.ucas_matched_applications.select(&:both_scheme?)`
will get us the UCASMatchedApplication objects we are after.
Then we make this method public
https://github.com/DFE-Digital/apply-for-teacher-training/blob/aee8187135fe126fb13bca580606cfa21a4a9460/app/models/ucas_matched_application.rb#L106-L112

This should get us ApplicationChoices in the correct cycle already.

Also there is potentially a scenario when a candidate applied for more than one course on both services.